dev-tools/docker/macosx_image/Dockerfile

```diff
@@ -29,14 +29,15 @@ RUN \
 RUN \
   mkdir -p /usr/local/sysroot-x86_64-apple-macosx10.11/usr && \
   cd /usr/local/sysroot-x86_64-apple-macosx10.11/usr && \
-  wget --quiet -O - https://s3-eu-west-1.amazonaws.com/prelert-artifacts/dependencies/usr-x86_64-apple-macosx10.11-2.tar.bz2 | tar jxf - && \
+  wget --quiet -O - https://s3-eu-west-1.amazonaws.com/prelert-artifacts/dependencies/usr-x86_64-apple-macosx10.11-3.tar.bz2 | tar jxf - && \
   wget --quiet -O - https://s3-eu-west-1.amazonaws.com/prelert-artifacts/dependencies/xcode-x86_64-apple-macosx10.11-1.tar.bz2 | tar jxf -
 
 # Build cctools-port
 RUN \
   git clone https://github.com/tpoechtrager/cctools-port.git && \
   cd cctools-port/cctools && \
   git checkout 895-ld64-274.2 && \
+  sed -i -e 's/autoconf/autoreconf -fi/' autogen.sh && \
```

Comment:
This is needed because in https://github.com/tpoechtrager/cctools-port/commit/07619027f8311fa61b4a549c75994b88739a82d8 an obscure version of `ldtools` was used to regenerate the `autoconf` files that are commited to the `895-ld64-274.2` branch in the `cctools-port` repo.